kalite/contentload/management/commands/init_content_items.py

```diff
@@ -56,14 +56,11 @@ def recurse_nodes(node, parent=""):
 
         node.pop("child_data", None)
 
-        child_availability = []
-
-        child_ids = [child.get("id") for child in node.get("children", [])]
+        total_files = 0
```

Comment:
Adds properties up the topic tree to show how many files are available for download.

Note: This is going to be removed from distributed, and in the content packs, initial file sizes will also be added.